wcc-contentful-app/app/helpers/wcc/contentful/app/section_helper.rb

```diff
@@ -25,4 +27,41 @@ def section_id(section)
     title = section.try(:bookmark_title) || section.try(:title)
     CGI.escape(title.gsub(/\W+/, '-')) if title.present?
   end
+
+  def markdown(text)
+    options = {
+      filter_html:     true,
+      hard_wrap:       true,
+      link_attributes: { rel: 'nofollow', target: '_blank' },
```

Comment:
We're always putting target '_blank' in these links, and I'm sure the issue will come up again where this isn't always the desired behavior. I'm not sure if this should be configurable some how, or if we need to define the behavior based on context, but anyways, I remember this being an issue in the past.

Reply:
We can override the `markdown` helper in the app, either by creating our own helper that extends from `WCC::Contentful::App::SectionHelper` or by copy-pasting code and tweaking it.